File: app/src/main/java/com/android/akl/bluetoothscreamer/MainView.java
package com.android.akl.bluetoothscreamer;

import java.util.List;

/**
 * Created by Mohamed Akl on 8/25/2018.
 */
public interface MainView {
    boolean isPrefAvailable(String selecteddPairedDevices);
    void addSharedPreferences(List<String> selectedDevices, List<String> notSelectedDevices);
    void setupSharedPreferences();
    void displayBTDevicesInMainActivity();
}

File: app/src/main/java/com/android/akl/bluetoothscreamer/MainView.kt
package com.android.akl.bluetoothscreamer

/**
 * Created by Mohamed Akl on 8/25/2018.
 */
interface MainView {
    fun isPrefAvailable(selecteddPairedDevices: String?): Boolean
    fun addSharedPreferences(selectedDevices: List<String?>?, notSelectedDevices: List<String?>?)
    fun setupSharedPreferences()
    fun displayBTDevicesInMainActivity()
}